tests/testthat/test-utils.R

```diff
@@ -0,0 +1,7 @@
+test_that("check_requires works", {
+  expect_true(check_requires("Imported packages", "cli"))
+  expect_error(
+    check_requires("Missing packages", "a_missing_package"),
+    regexp = "Missing packages requires"
+  )
+})
```

Comment:
Updated the CHANGELOG. There are already these tests unless you want to check specific packages?

Reply:
I overlooked on the first review, apologies!